docs/download/repositories.md

```diff
@@ -65,7 +65,7 @@ The European Genome-Phenome Archive ([EGA](https://ega-archive.org/)) is co-mana
 | Repo Code        | `ega`                                                                                                                                  |
 
 ### Obtaining Data Access
-Follow the procedure outlined at the [DACO page](https://icgc.org/daco). Once approved by ICGC DACO, you will need to contact EGA to have your EGA account set up.
+Follow the procedure outlined at the [DACO page][1]. Once approved by ICGC DACO, you will need to contact EGA to have your EGA account set up.
```

Comment:
Let's change to:

To obtain access to Controlled Data, learn more about the [DACO application process, ](https://docs.icgc-argo.org/docs/data-access/daco/applying)or [start a DACO application here](https://daco.icgc-argo.org/). Once approved by the ICGC DACO, you will need to contact EGA to have your EGA account set up.